app/proguard-rules.pro

```diff
@@ -73,8 +73,8 @@
 -keep @interface pm.gnosis.heimdall.data.adapters.DecimalNumber
 -keep @interface pm.gnosis.heimdall.data.adapters.BigDecimalNumber
 
--keep interface io.gnosis.data.backend.dto.** { *; }
--keep class io.gnosis.data.backend.dto.** { *; }
```

Comment:
We still have CoinBalancesDto. Shouldn't we keep this until CoinBalancesDto and SafeInfoDto have been removed?

Reply:
SafeInfoDto we don't need, and assets still work, because we have this rule

Reply:
```